addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -111,22 +111,64 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+// Try to update the scrollbar based on the current context (modal state) 
+// and only if the modal overflowing has changed
+
+function _updateScrollbar(event) {
```

Comment:
Not sure how that convention is followed in other teams but in general I would avoid naming a variable `event` and instead use `ev`. Chrome has the wonderful idea to name one of its magic global variable `event` so this leads to code errors if a dev forget to declare the variable event (like you did here) but still uses it: it will work on Chrome but crash on Firefox :grimacing:

Reply:
:+1: